glad/parse.py

```diff
@@ -74,8 +74,13 @@ def comment(self):
     @property
     def types(self):
         if self._types is None:
-            self._types = [Type(element) for element in
-                           self.root.find('types').iter('type')]
+            elements = self.root.find('types').iter('type')
```

Comment:
Modifying the parser is incorrect here. I don't want the parser behaviour to change.